test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -570,11 +583,12 @@ public async Task When_error_handler_closes_event_source_no_reconnect_attempt_is
             Assert.Equal(1, handler.GetRequests().Count());
         }
 
+        [Fact]
         public async Task Connection_closed_before_threshold_gets_increasing_backoff_delay()

```

Comment:
The fixes in this test and the next test are unrelated to this PR, but these tests were broken and always had been. We just didn't see it because they weren't actually being run (due to the lack of `[Fact]`).